### PR TITLE
Added option to get Classes at specific time

### PR DESCRIPTION
**Pull request by fraret**

### CODE REVIEW

File: inc/Classes.php [hunk 1]
```diff
@@ -37,9 +42,16 @@ public static function getCurrentClasses() {
           '.($isSignedIn ? 'u_s.subject_id IS NULL,
           ' : '').'s.friendly_name ASC';
     $query = $con->prepare($sentence);
-    
-    if (!$query->execute(($isSignedIn ? ['user_id' => Users::getUserId()] : [])))
-      return false;
+
+    if(!$unix_time) {
```
Comment (avm99963) [inline inc/Classes.php]: Aquí posaria un espai després de l'if

File: inc/Classes.php [hunk 2]
```diff
@@ -37,9 +42,16 @@ public static function getCurrentClasses() {
...
+      $unix_time = time();
+    }
+
+    $query_params = ['unix_time' => $unix_time];
+    if($isSignedIn) $query_params['user_id'] = Users::getUserId();
```
Comment (avm99963) [inline inc/Classes.php]: I aquí també

**Comment by avm99963** [comment]
Excepte pels canvis menors d'espaiat, LGTM, genial! 🎉🎉

**Review by avm99963 — APPROVED** [review]
Excepte pels canvis menors d'espaiat, LGTM, genial! 🎉🎉

**Comment by fraret** [comment]
Latest commit message is wrong, the method has been changed to GET.
Spacing has been improved.

When merging, if possible squash and give it a meaningful commit message

**Review by avm99963 — APPROVED** [review]
LGTM, I'll squash all the commits and merge them!

**Comment by avm99963** [comment]
Ho sento per canviar el master tota l'estona fent amendments al commit del merge. No aconseguia fer que el commit se t'atribuís a tu :(

**Pull request closed without merging**